File: nginx/MASZ/src/app/models/GuildMotd.ts
import { DiscordUser } from "./DiscordUser";

export interface GuildMotd {
    id: number;
    guildId: string;
    message: string;
    showMotd: boolean;
    userId: string;
    createdAt: Date;
}

export interface GuildMotdView {
    motd: GuildMotd;
    creator?: DiscordUser;
}

File: backend/MASZ/Models/Database/GuildMotd.cs
namespace MASZ.Models
{
    public class GuildMotd
    {
        public int Id { get; set; }
        public ulong GuildId { get; set; }
        public ulong UserId { get; set; }
        public DateTime CreatedAt { get; set; }
        public string Message { get; set; }
        public bool ShowMotd { get; set; }
    }
}
